nixos/tests/devpi-server.nix

```diff
@@ -0,0 +1,35 @@
+import ./make-test-python.nix ({ pkgs, ...}:
```

Comment:
Don't forget to also bind your NixOS test to the devpi-server package:

```nix
  passthru.tests.devpi-server = nixosTests.devpi-server;
```

in pkgs/development/tools/devpi-server/default.nix